### ISSUE TITLE
Items not being removed from stage

### ISSUE THREAD

**Issue by happybandit360** [Darkhax-Minecraft/ItemStages#51]
I am trying to remove items from being staged after staging whole mods and I can seem to get it working.  Here is one example of a script I'm using
```
#priority 3400

import crafttweaker.item.IIngredient;
import crafttweaker.item.IItemStack;
import mods.zenstages.Stage;
import mods.zenstages.ZenStager;


// Unstage Powders
mods.ItemStages.removeItemStage(<enderio:item_material:30>);
mods.ItemStages.removeItemStage(<enderio:item_material:31>);
```

I'm not getting any errors, but it's also not unstaging any of the items I am using with the mods.ItemStages.removeItemStage method. Any idea what I'm doing wrong here?

**Comment by Darkhax** [comment]
You are potentially removing the items from the stage or staging them again afterwards. Check with the log file to make sure the remove action is the last one. It is also possible that this is an issue that specifically affects Ender IO items, which won't be fixed on 1.12.2.

**Comment by happybandit360** [comment]
Priority was the problem. Thanks
